PopupMenu/PopupMenu.js

```diff
@@ -37,7 +39,11 @@ const PopupMenuBase = kind({
 		className: 'popupMenu'
 	},
 
-	render: ({children, closeButton, css, noAnimation, onClose, onHide, open, title, ...rest}) => {
+	computed: {
+		bodyClassName: ({css, orientation}) => orientation === 'horizontal' ? css.horizontalBody : css.body
```

Comment:
How do you feel about having this class just be `.horizontal` and then we can use a `.horizontal .body` selector to target the necessary no-wrap rule. This would mean changing this prop to `className` from `bodyClassName`.

Also, we could set `orientation` to default to "horizontal" so the other PR isn't needed: https://github.com/enyojs/webos-auto-home/pull/38/files#diff-32b7ef32f74375c7a9715daa2e883fd7R60